ci-jobs/build.yml

```diff
@@ -9,4 +9,4 @@ jobs:
           addChangeLog: false
   - template: ./templates/build.yml
     parameters:
-      name: 'macOS_10_14'
\ No newline at end of file
+      name: 'macOS_10_15'
```

Comment:
the pipeline for regular push checks is at https://github.com/appium/WebDriverAgent/blob/master/.azure-pipelines.yml